doc/whats_new.rst

```diff
@@ -31,6 +31,8 @@ Changelog
 
     - Add support for the University of Maryland KIT system by `Christian Brodbeck`_
 
+    - Add the `--filterchpi` option to `mne browse_raw`, by `Felix Raimundo`_
```

Comment:
Nah don't bother to change the other refs in `whats_new.rst`, let's just get it right from here on. And I checked, it needs to be:
```
... in :ref:`mne browse_raw <gen_mne_browse_raw>` by ...
```